recipes/cuda-compiler/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: cuda-compiler
+  version: 12.0.0
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_documentation/LICENSE.txt
+  sha256: 5db25d4fd138013b563f9a3d1d87f7de7df1dac014fd4cccdfbb3435a5cff761
+
+build:
+  number: 0
+  skip: true  # [osx]
+
+requirements:
+  run:
+    - cuda-cuobjdump 12.0.76
```

Comment:
Will these component versions always be in sync? If so, should we use a variable that is defined once?

Reply:
No, all components are versioned independently.